new-feature.txt

```diff
@@ -0,0 +1,3 @@
+This file was created in the feature1 branch.
```

Comment:
It's really good that you have decided to add this line of code.

Reply:
Thanks!